packages/app-bridge/src/react/useAssetChooser.ts

```diff
@@ -1,16 +1,26 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
+import {useEffect, useState} from 'react';
 import type { AppBridgeBlock } from '../AppBridgeBlock';
-import type { Asset, AssetChooserOptions } from '../types';
+import type { Asset, AssetChooserOptions, CommandResponse } from '../types';
 
 type UseAssetChooserType = {
     openAssetChooser: (callback: (selectedAsset: Asset[]) => void, options: AssetChooserOptions) => void;
     closeAssetChooser: () => void;
 };
 
 export const useAssetChooser = (appBridge: AppBridgeBlock): UseAssetChooserType => {
+    const [assetChooser, setAssetChooser] = useState<CommandResponse['AssetChooser.Open'] | null>(null);
+
     return {
-        openAssetChooser: appBridge.openAssetChooser.bind(appBridge),
-        closeAssetChooser: appBridge.closeAssetChooser.bind(appBridge),
+        openAssetChooser:  async () => {
+            const dispatchResponse = await appBridge.dispatch('AssetChooser.Open');
+            setAssetChooser(dispatchResponse);
+        },
+        closeAssetChooser: async () => {
+            if(assetChooser) {
```

Comment:
Please setup your IDE for ESLint/Prettier, lot of code in not respecting the coding guidelines 🙈

Reply:
Sorry about that. This is resolved 🙈